changelogs/fragments/parted_negative_numbers.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

```suggestion
minor_changes:
```
I would rather see this as a new feature. Or did the module document that you can use negative values?

---

Bug reported pointed out that "all formats supported by parted" should include negative numbers (relative to disk end) too, but module documentation mentions only "distance from start of the disk". I'll include documentation change in PR.